libraries/helpers.rb

```diff
@@ -74,5 +66,12 @@ def home_basedir
         '/home'
       end
     end
+
+    # Returns the type of the public key or rsa
+    #
+    # @return [String]
+    def pubkey_type(pubkey)
+      %w(ed25519 ecdsa dss rsa dsa).filter { |kt| pubkey.split.first.include? kt }.first || 'rsa'
```

Comment:
Just stick some braces around the `include?` argument and I think we can call this done for now 😀

Reply:
At the risk of sounding a bit stupid. Do you mean curly braces around the `kt` in this case?

Like this:

```%w(ed25519 ecdsa dss rsa dsa).filter { |kt| pubkey.split.first.include? { kt } }.first || 'rsa'```

If my assumption is correct, what does that achieve? I never used include with the braces before and `kt` is already an interator right so it does not need to be executed before hand?

Reply:
Aaah, I just checked your previous example and I suppose you mean parantheses here, so I will go with that for now. If you do mean curly braces then please let me know because I'd like to learn the why :)

Reply:
Yes I did mean parantheses, apologies there.